extensions/amp-fit-text/1.0/README.md

```diff
@@ -0,0 +1,218 @@
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-fit-text-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  bento-fit-text {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
+```
+
+**Container type**
+
+The `bento-fit-text` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children (slides) via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties):
+
+```css
+bento-fit-text {
+  height: 100px;
+  width: 100%;
+}
+```
+
+#### Accessibility considerations of overflowing content
+
+While overflowing content is _visually_ truncated to fit the container, note that it's still present in the document. Do not rely on the overflow behaviour to simply "stuff" large amounts of content in your pages - while visually it may look appropriate, it may lead to the page becoming overly verbose to users of assistive technologies (such as screen readers), as for these users all the truncated content will still be read/announced in full.
+
+#### Attributes
+
+##### Media Queries
+
+The attributes for `<bento-fit-text>` can be configured to use different
+options based on a [media query](./../../../docs/spec/amp-html-responsive-attributes.md).
+
+##### `min-font-size`
+
+Specifies the minimum font size as an integer that the `bento-fit-text` can use.
+
+##### `max-font-size`
+
+Specifies the maximum font size as an integer that the `bento-fit-text` can use.
+
+### Preact/React Component
+
+The examples below demonstrate use of the `<BentoFitText>` as a functional component usable with the Preact or React libraries.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-fit-text
+```
+
+```javascript
+import React from 'react';
+import {BentoFitText} from '@ampproject/bento-fit-text/react';
+import '@ampproject/bento-fit-text/styles.css';
+
+function App() {
+  return (
+    <BentoFitText>
+      Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque
+      inermis reprehendunt.
+    </BentoFitText>
+  );
+}
+```
+
+[/example]
+
+#### Layout and style
+
+**Container type**
+
+The `BentoFitText` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children (slides) via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties). These can be applied inline:
+
+```jsx
+<BentoFitText style={{width: '300px', height: '100px'}}>
```

Comment:
Ok to leave as is but consider removing 'px' and making these numbers. They'll be rendered as pixel values by default.

```suggestion
<BentoFitText style={{width: 300, height: 100}}>
```

Reply:
Looks a lot cleaner, thanks!